server/build/webpack.js

```diff
@@ -45,7 +45,7 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
     new webpack.optimize.CommonsChunkPlugin({
       name: 'commons',
       filename: 'commons.js',
-      minChunks: pages.length
+      minChunks: pages.length + 1
```

Comment:
@arunoda you are right, but for out goal need get at least 2 chunks so `require.ensure` start working properly. I'll add a simple check for it.